allensdk/experimental/nwb/visbeh_dev.py

```diff
@@ -0,0 +1,69 @@
+from pynwb import NWBFile, NWBHDF5IO, TimeSeries
+import os
+import pandas as pd
+from visual_behavior.translator.foraging2 import data_to_change_detection_core
+import datetime
+from allensdk.experimental.nwb.stimulus import VisualBehaviorStimulusAdapter
+
+def add_epochs(nwbfile, ts_list, core_data):
+
+    # REFACTOR to stimulus?
```

Comment:
I think refactoring to stimulus is a good idea. Then you could use your methods for getting time either from sync or from the pkl file, since for ophys and ecephys we'll want to use sync timestamps.

Reply:
Yeah that worked nicely.